### ISSUE TITLE
Define DevPool payment cases

### ISSUE THREAD

**Issue by sergfeldman** [ubiquity/research#11]
**This research is part of a larger research on launching Ubiquity Cards** 
The purpose is to define the exact payment cases to communicate with payment providers. 

### Crypto

Steps 
- Partner project tops up the DevPool wallet.
- Once the bounty is done, the bounty hunter provides his/her crypto wallet and claims the bounty.

**Safe apps for payment**
As Ubiquity uses Safe, we should research Safe apps for payments
https://app.safe.global/

https://github.com/bh2smith/safe-airdrop 
https://llamapay.io/
https://sablier.finance/
https://www.superfluid.finance/
It seems Safe supports payment streams because most of the payment apps are related to streams.

### Fiat

Incoming international fiat transfers can be associated with additional fees for receivers.
This is applicable to the cards as well as banking accounts, and do not  applicable to non-banking transfers (Western Union, etc)

**Option 1**

Process view
- Partner project transfers uAD to Ubiquity.
- Once the bounty is done, the bounty hunter provides his/her payment details 
(desired fiat currency, card number/IBAN/SWIFT code, valid name, personal identifier, etc).
- Ubiquity sells uAD to get fiat.  
- Ubiquity pays the bounty in fiat by the provided payment details.

Legal view
- The company that issues Ubiquity cards is registered in country A (ie Singapore or Malta)
- The freelancer that gets the Ubiquity card is a resident of country B (ie South Korea or France)
- The payment to the Ubiquity card is performed from the company registered in country C (ie British Virgin Islands)
- The freelancer spends money via Ubiquity cards in country D (ie Thailand or Turkey) 

**Option 2**
- Partner project somehow (by uAD?) transfers money to Ubiquity.
- Once the bounty is done, the bounty hunter generated a fiat payment request.
- Ubiquity pays the bounty in fiat by the payment request (potentially, can be automated).

Payment request abilities popular among freelancers 
https://www.payoneer.com/solutions/send-payment-request/
https://wise.com/help/articles/2WvlZST6DiDMUBhyl1N4zM/how-do-i-request-money

### Other payment methods

**Pay by fiat gift cards**

Gift cards are purchased for crypto.
Then gift cards can be spent as fiat.

For example
https://www.bitrefill.com/
https://tokenstore.io/
https://coingate.com/gift-cards

**Binance services**

Internal transfer between Binance accounts in crypto and several fiat currencies via API
https://www.binance.com/en/support/faq/how-to-make-internal-transfer-on-binance-360037037312 
https://developers.binance.com/docs/binance-pay/api-wallet-transfer

Crypto gift cards via API
https://www.binance.com/en/blog/gift-card/unlock-business-opportunities-with-binance-gift-card-solution-2745950951180064643
https://www.binance.com/en/gift-card
https://www.binance.com/en/support/faq/how-to-create-dual-token-binance-gift-cards-via-api-92e59df4caf649409a256c3e6b1e1739

Binance pay - crypto and several fiat currencies (as I understand, currently manually)
https://merchant.binance.com/en/products/payout
https://www.binance.com/en/support/faq/how-to-send-cash-using-binance-pay-72dc08f042f84f92824ea9a86a491917

**Payments inside Ubiquity wallet**

User experience like in Binance - usable transfers inside the Ubiquity wallet.
The wallet is connected to multiple payment methods - Web3 wallets, bank cards, wire transfers, etc.

Quick implementation can be done with white-label solutions (not Web3 wallets)

White-label platform with different asset classes
Great for a truly all-in-one solution.
https://www.bitpanda.com/en/tech

White-label wallets 
Credit cards and everything related should be created separately.
https://sdk.finance/digital-wallet-solution/
https://youtap.com/products/card-management-and-processing/white-label-digital-wallet-and-card/

### Separate card issuing from card payments

For the Web3 wallets we use approach "Bring your own" and do not insist on use, for example, Ubiquity wallet (based on Safe).
For the bank cards we want to offer freelancers our own cards.
It is not clear will it be possible to use "Bring your own" for the cards.

A good direction is to facilitate use of the Ubiquity cards not only for getting paid from the DevPool, but getting paid from other places as well.
The strategy is similar to the Payoneer pisition for the Upwork and other freelance marketplaces.
Such approach is not only makes Ubiquity cards nore attractive for freelancers, but also is helpful for the legal purposes.

A good option at the moment is to open a non-crypto company in a legit country.
It should be a separate company not related to crypto (for example DevPool company).
It will increase chances of issuing cards by top card issuers for freelancers - Wise, Payoneer, and potentially Stripe.

Roles of the companies:
- DevPool - issues cards for bounty hunters, organizes loyalty program ie cashback, discounts, etc.
- Ubiquity - gets paid by DevPool partners, and pays to bounty hunters either in crypto or in fiat.
Ubiquity itself can pay to the cards to avoid taxes for the new DevPool company.

This way not only helps to issue cards with leading card issuers but also protects us from the risks of a possible negative change of the regulation of card issuing by crypto companies. 

In the middle of 2024, the situation can significantly change with the European MiCA crypto regulation.
Stablecoins are regulated by the MiCA.

In a positive case, if Ubiquity Dollar approved by regulators, we can start issuing cards from Ubiquity itself.
In a negative case, if Ubiquity Dollar does not approved by regulators, we will have to look for the card issuing services outside of Europe.
Card issuers can also change the requirements to the crypto companies.

The proposal is to move in 2 steps:
- Have a card issuing solution for 2023 - first half of 2024.
- Monitor the MiCA implementation and potentially change the card issuing solution to the end of 2024.

As a next step
**[Check other available non-custodial wallets with bank cards](https://github.com/ubiquity/research/discussions/16)**

**Comment by 0x4007** [comment]
This should not be an issue. This doesn't look like a project specification. This should probably be in a document in the repository but I'm unsure what this is a deliverable for.
